--- a/arch/mips/cavium-octeon/executive/cvmx-helper-npi.c	2024-02-21 11:22:24.280205206 +1100
+++ b/arch/mips/cavium-octeon/executive/cvmx-helper-npi.c	2024-02-21 11:23:12.772725096 +1100
@@ -49,7 +49,7 @@
 int __cvmx_helper_npi_probe(int interface)
 {
 #if CVMX_PKO_QUEUES_PER_PORT_PCI > 0
-	if (OCTEON_IS_MODEL(OCTEON_CN38XX) || OCTEON_IS_MODEL(OCTEON_CN58XX))
+	if (OCTEON_IS_MODEL(OCTEON_CN38XX) || OCTEON_IS_MODEL(OCTEON_CN58XX) || OCTEON_IS_MODEL(OCTEON_CN7XXX))
 		return 4;
 	else if (OCTEON_IS_MODEL(OCTEON_CN56XX)
 		 && !OCTEON_IS_MODEL(OCTEON_CN56XX_PASS1_X))
